comms/rabbitmq.go

```diff
@@ -683,19 +683,19 @@ func (rmq rabbitMQComms) ListenForGroupTasks(groupName string, dereg chan bool)
 		for d := range msgs {
 
 			// Unmarshal into BaseTaskMessage to determine type
-			var base_msg tasks.BaseTask
-			err = json.Unmarshal(d.Body, &base_msg)
+			var baseMsg tasks.BaseTask
+			err = json.Unmarshal(d.Body, &baseMsg)
 			// TODO(mierdin): Need to handle this error
 
 			log.Debugf("Agent task received: %s", d.Body)
 
 			// call agent task method based on type
-			switch base_msg.Type {
+			switch baseMsg.Type { // TODO: remove switch?
 
 			// This has been removed, as I am moving away from using queues that use the group name.
```

Comment:
I may actually want to re-use this mechanism. Can you replace this comment with a TODO that just calls this out?